src/components/guide-tooltip/GuideTooltip.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import Tooltip, { TooltipTheme } from '../tooltip';
+import Button from '../button';
+import messages from './messages';
+
+import './GuideTooltip.scss';
+
+type TooltipProps = Omit<JSX.LibraryManagedAttributes<typeof Tooltip, Tooltip['props']>, 'text' | 'theme'>;
+
+type Props = TooltipProps & {
+    body: React.ReactNode;
+    title?: React.ReactNode;
```

Comment:
many are optional. What happens if they all are left off? Do we have an example of that?

Reply:
This example is the first screenshot. Only had body